framework_tvb/tvb/core/services/project_service.py

```diff
@@ -570,9 +576,10 @@ def _remove_project_node_files(self, project_id, gid, skip_validation=False):
                     new_op = dao.store_entity(new_op)
                     to_project = self.find_project(links[0].fk_to_project).name
                     full_path = h5.path_for_stored_index(datatype)
-                    self.structure_helper.move_datatype(datatype, to_project, str(new_op.id), full_path)
+                    new_op_folfer = self.structure_helper.move_datatype(datatype, to_project, str(new_op.id), full_path)
                     datatype.fk_from_operation = new_op.id
                     datatype.parent_operation = new_op
+                    h5.store_view_model(view_model, new_op_folfer)
```

Comment:
This new storage of the VM will work, and mostly do the job.
But we could also just move the original VM H5 file, isn't it ?
I fear that by calling store_vie_model again we might change some attributes (e.g. `create_data`). Could this be true?